src/docs/development/ios-project-migration.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Xcode 11.4 Support
+description: How to migrate existing Flutter iOS projects to Xcode 11.4.
+---
+
+{{site.alert.note}}
+  You might be directed to this page if Flutter detects that your project
+  doesn't support Xcode 11.4.
+{{site.alert.end}}
+
+To develop Flutter apps for iOS, you need a Mac with Xcode installed.
+Xcode 11.4 changed the way frameworks are linked and embedded, and
+you may [experience issues switching between iOS devices and simulators][].
+Flutter v1.15.3 and later will automatically migrate your Xcode project.
+
+If you need to manually upgrade your project, use the following steps:
+<ol markdown="1">
```

Comment:
FWIW, this is barely documented but you can unbreak the weird markdown ordered list thing with big blocks if you 4 space indent everything i.e. 

```
1. some thing

    `some long code section which would have made a mess`
1. this should be item 2 and should work now
```

Reply:
Interesting!  I don't know if @sfshaza2 tried that, but the `ol` markdown was how she got  https://github.com/flutter/website/blob/master/src/docs/development/add-to-app/ios/project-setup.md to work.  She asked me to do it here so I'll stick with that for now, but I can follow up and experiment when she's back in the office next week.